src/main/java/qna/domain/Answer.java

```diff
@@ -47,11 +49,12 @@ public Answer(Long id, User writer, Question question, String contents) {
     }
 
     public boolean isOwner(User writer) {
-        return this.user.getId().equals(writer.getId());
+        return this.user.equals(writer);
     }
 
     public void toQuestion(Question question) {
-        this.question.setId(question.getId());
+        this.question = question;
+        question.getAnswers().add(this);
```

Comment:
```suggestion
        this.question.addAnswer(this);
```
question.getAnswers() 으로 꺼내서 작업하지 말고 
question 객체에 작성되어있는 메서드를 활용하면 좋을 거 같아요 ㅎㅎ

Reply:
AddAnswer 를 활용하는 것으로 변경했습니다.